src/components/CityWeatherShortcut/CityWeatherShortcut.styled.tsx

```diff
@@ -0,0 +1,53 @@
+import styled from 'styled-components';
+
+const Container = styled.div`
+  width: 160px;
+  min-width: 160px;
```

Comment:
**question** is this `min-width` located here for solving too long cityNames? If so i think it wouldn't be symmetrical and in my opinion it won't look great

Reply:
but then if cityName is too long it will overflow so we have to find a better solution. On map it is great solution as there is no need for symmetry

Reply:
text-overflow: ellipsis ?